lib/chef/resource/chocolatey_source.rb

```diff
@@ -35,6 +35,12 @@ class ChocolateySource < Chef::Resource
       property :priority, Integer, default: 0,
                description: "The priority level of the source."
 
+      property :user, String,
+               description: "The username to authenticate to the source"
+
+      property :password, String,
```

Comment:
Can you add introduced properties here for 14.6 since that's the version we'll ship this in. Here's an example: https://github.com/chef/chef/blob/429345d5a75782da0d6544bb3f60fa89ce964b94/lib/chef/resource/ifconfig.rb#L48